engine/runtime-integration-tests/src/test/java/org/enso/compiler/test/IfThenElseTest.java

```diff
@@ -11,93 +10,80 @@
 import com.oracle.truffle.api.library.ExportLibrary;
 import com.oracle.truffle.api.library.ExportMessage;
 import java.io.ByteArrayOutputStream;
-import org.enso.test.utils.ContextUtils;
-import org.graalvm.polyglot.Context;
+import org.enso.test.utils.ContextRule;
 import org.graalvm.polyglot.PolyglotException;
 import org.graalvm.polyglot.Value;
 import org.hamcrest.Matchers;
 import org.hamcrest.core.AllOf;
 import org.junit.After;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Ignore;
 import org.junit.Test;
 
 public class IfThenElseTest {
-  private static Context ctx;
   private static final ByteArrayOutputStream MESSAGES = new ByteArrayOutputStream();
 
-  @BeforeClass
-  public static void initEnsoContext() {
-    ctx = ContextUtils.defaultContextBuilder().out(MESSAGES).err(MESSAGES).build();
-    assertNotNull("Enso language is supported", ctx.getEngine().getLanguages().get("enso"));
-  }
+  @ClassRule public static final ContextRule ctxRule = ContextRule.createWithCapturedOut(MESSAGES);
 
   @After
   public void cleanMessages() {
     MESSAGES.reset();
```

Comment:
`@After` would be unnecessary if there was `ContextRule.stdOut` and/or `ContextRule.stdErr` or both.

Reply:
Done in bdfaa6bfaaa3052d7baac7f46074f530ff2ed974.